pkg/v1/cli/command/plugin/doc.go

```diff
@@ -19,14 +21,20 @@ var (
 )
 
 func init() {
-	genDocsCmd.Flags().StringVarP(&docsDir, "docs-dir", "d", core.DefaultDocsDir, "destination for docss output")
+	genDocsCmd.Flags().StringVarP(&docsDir, "docs-dir", "d", core.DefaultDocsDir, "destination for docs output")
 }
 
 var genDocsCmd = &cobra.Command{
 	Use:    "generate-docs",
 	Short:  "Generate Cobra CLI docs for all subcommands",
 	Hidden: true,
 	RunE: func(cmd *cobra.Command, args []string) error {
+		if docsDir == "" {
+			docsDir = core.DefaultDocsDir
+		}
+		if _, err := os.Stat(docsDir); err != nil {
```

Comment:
Here too I think we should check that `docsDir` is a directory

Reply:
yes thats correct.